spec/lib/appsignal_spec.rb

```diff
@@ -92,18 +92,23 @@
 
       context "when allocation tracking and gc instrumentation have been enabled" do
         before do
+          GC::Profiler.stub(:enable)
```

Comment:
RSpec 3 (which I hope we can upgrade to) has the preferred style of `expect|allow(object).to be_something` rather than using the core extensions allowing `object.should be_something`

So I'm expecting to switch the entire test suite to this style over time.